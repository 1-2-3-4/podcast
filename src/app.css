.app-container {
  font-family: Helvetica, sans-serif;
  max-width: 600px;
  margin: 67px auto;
}

.app-nav {
  min-height: 65px;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  background-color: #202020;
  color: #ececec;
  padding: 0 10px;
  z-index: 100;
}