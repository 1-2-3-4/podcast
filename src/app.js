import * as React from 'react'

import { AudioProvider } from './components/audio-provider/audio-provider'
import { AudioList } from './components/audio-list/audio-list'
import { NowPlaying } from './components/now-playing/now-playing'

import './app.css'

export const App = () => {
  return (
    <>
      <nav className="app-nav">
        <h1>Playlist</h1>
      </nav>
      <div className="app-container">
        <AudioProvider>
          <NowPlaying/>
          <AudioList/>
        </AudioProvider>
      </div>
    </>
  );
}

export default App
