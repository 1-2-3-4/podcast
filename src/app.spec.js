import * as React from 'react'
import { shallow } from 'enzyme'

import { App } from './app'

describe('App', () => {
  describe('when it renders', () => {
    let wrapper

    beforeAll(() => {
      wrapper = shallow(<App/>)
    })

    it('matches snapshot', () => {
      expect(wrapper.getElement()).toMatchSnapshot()
    })
  })
})
