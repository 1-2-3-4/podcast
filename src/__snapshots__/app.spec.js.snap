// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App when it renders matches snapshot 1`] = `
<React.Fragment>
  <nav
    className="app-nav"
  >
    <h1>
      Playlist
    </h1>
  </nav>
  <div
    className="app-container"
  >
    <AudioProvider>
      <NowPlaying />
      <AudioList />
    </AudioProvider>
  </div>
</React.Fragment>
`;
